File: app/src/main/java/com/juniperphoton/myersplash/utils/SingleMediaScanner.java
package com.juniperphoton.myersplash.utils;

import android.content.Context;
import android.content.Intent;
import android.net.Uri;

import java.io.File;

public class SingleMediaScanner {
    public static void sendScanFileBroadcast(Context context, File file) {
        Intent intent = new Intent(Intent.ACTION_MEDIA_SCANNER_SCAN_FILE);
        intent.setData(Uri.fromFile(file));
        context.sendBroadcast(intent);
    }
}

File: app/src/main/java/com/juniperphoton/myersplash/utils/SingleMediaScanner.kt
package com.juniperphoton.myersplash.utils

import android.content.Context
import android.content.Intent
import android.net.Uri
import java.io.File

object SingleMediaScanner {
    fun sendScanFileBroadcast(ctx: Context, file: File) {
        var intent = Intent(Intent.ACTION_MEDIA_SCANNER_SCAN_FILE)
        intent.data = Uri.fromFile(file)
        ctx.sendBroadcast(intent)
    }
}